src/framework/MOM_domains.F90

```diff
@@ -1566,6 +1569,28 @@ subroutine MOM_domains_init(MOM_dom, param_file, symmetric, static_memory, &
     endif
   endif
 
+  global_indices(1) = 1 ; global_indices(2) = int(MOM_dom%niglobal/2)
+  global_indices(3) = 1 ; global_indices(4) = int(MOM_dom%njglobal/2)
+  xhalo_zap2 = int(MOM_dom%nihalo/2)
```

Comment:
Unfortunately this does not work. I have to choose the MOM_dom%nihalo/dl otherwise the checks for downsampled field size complain that it does not match the expected size of original_dszi/dl .